src/main/java/htsjdk/variant/bcf2/BCF2Codec.java

```diff
@@ -150,6 +150,10 @@ public FeatureCodecHeader readHeader( final PositionalBufferedStream inputStream
 
             if ( bcfVersion.getMajorVersion() != ALLOWED_MAJOR_VERSION )
                 error("BCF2Codec can only process BCF2 files, this file has major version " + bcfVersion.getMajorVersion());
+
+            // TODO: fixing this breaks GATK GenomicsDB integration/tests
```

Comment:
Agreed. It needs to be fixed, but that will go into the next PR. This is one several places in this PR where I added notes like this for things like this.